### PR TITLE
feat: add `#get` utility

### PR DESCRIPTION
**Pull request by nikku**
This adds `get` as a partner to the recently added `set` util.

### CODE REVIEW

File: lib/index.d.ts [hunk 1]
```diff
@@ -65,6 +65,15 @@ export function assign<T, U, V, W>(target: T, source1: U, source2: V, source3: W
...
 export function assign<T>(target: T, ...sources: any[]): T;
 
+/**
+ * Gets a nested property of a given object, with an optional default value.
+ *
+ * @param target The target of the get operation.
+ * @param path The path to the nested value.
+ * @param value The default value to return if the property does not exist.
+ */
+export function get(target: any, path: (string|number)[], value?: any): any;
```
Comment (philippfromme) [inline lib/index.d.ts:75]: `value` should be `defaultValue`.

File: lib/object.js [hunk 2]
```diff
@@ -60,6 +60,29 @@ export function set(target, path, value) {
...
+ *
+ * @param {Object} target The target of the get operation.
+ * @param {(string|number)[]} path The path to the nested value.
+ * @param {any} [defaultValue] The value to return if no value exists.
+ */
+export function get(target, path, defaultValue) {
+
+  var currentTarget = target;
+
+  forEach(path, function(key, idx) {
```
Comment (philippfromme) [inline lib/object.js:75]: `idx` is not used anywhere.

**Review by philippfromme — CHANGES_REQUESTED** [review]
Two small things, otherwise fine. 👍

File: test/object.spec.js [hunk 3]
```diff
@@ -439,4 +440,31 @@ describe('object', function() {
...
+    it('should return object property', function() {
+      expect(get({}, ['a'])).to.equal(undefined);
+      expect(get({}, ['a'], 'FOO')).to.equal('FOO');
+
+      expect(get({ a: 0 }, ['a'])).to.equal(0);
+      expect(get({ a: 0 }, ['a'], 1)).to.equal(0);
+
+      expect(get({ a: { b: 0 } }, ['a', 'b'])).to.equal(0);
+      expect(get({ a: { } }, ['a', 'b'], 1)).to.equal(1);
+    });
```
Comment (barmac) [inline test/object.spec.js:455]: Do we want it to be error-proof?

```suggestion
      expect(get({ a: null }, ['a', 'b'], 1)).to.equal(1);
    });
```

File: lib/object.js (hunk 2 above)
Comment (nikku) [reply lib/object.js]: https://github.com/bpmn-io/min-dash/pull/19/commits/796ccc6a1f33f0ebcf03e15bda75643c238d17df

File: test/object.spec.js (hunk 3 above)
Comment (philippfromme) [reply test/object.spec.js:462]: I'd say yes. I don't expect any errors to be thrown by this function.

**Review by philippfromme — APPROVED** [review]
Lovely! 🥇

**Comment by philippfromme** [comment]
@nikku I will release this as v3.7.0 today so we can start using it right away.

**Pull request merged**